A16-tcp-user-timeout.md

```diff
@@ -18,13 +18,18 @@ TCP_USER_TIMEOUT is a socket option that has been available on Linux since kerne
 At the time this proposal was written, gRPC Core faced an issue where data could be stuck for a very long time at the gRPC TCP layer. For cases where the network goes down, and the socket cannot accept more data for writing, an RPC could be blocked for a very long time. Even if the deadline expired on such an RPC, gRPC Core was incapable of shutting down the RPC, because the TCP layer would still have references to user data. Setting TCP_USER_TIMEOUT to a reasonable value would put a bound on the maximum time the TCP layer would wait for TCP packets to be acknowledged, and close the connection if the packets are not acknowledged in time. This would end up releasing references to user data at the gRPC TCP layer, thereby allowing the RPC to fail in a timely manner.
 
 ### Related Proposals:
-N/A
+[Client-side Keepalive](https://github.com/grpc/proposal/blob/master/A8-client-side-keepalive.md)
```

Comment:
You might consider making these links relative (i.e., just "A8-client-side-keepalive.md"). The are easier to read and work properly in more environments.